src/components/components/index.js

```diff
@@ -6,9 +6,20 @@ import useDidMount from "../useDidMount";
 export default ({ loading, components }) => {
   const [hasMounted] = useDidMount();
 
+  // Copy and sort by the GitHub issue title
+  const sorted = [...components];
+  sorted.sort((a, b) => {
```

Comment:
Maybe just use `localeCompare`. It's more compact, and if for some unforeseen reason we change "Relay Servers" to "Đérṕ", it'll still sort correctly.

```jsx
components.sort((a, b) => a.title.localeCompare(b.title))
```

You can also copy an array by just using `arr.slice()`, so this whole block can be shortened to:

```jsx
const sorted = components.slice().sort((a, b) => a.title.localeCompare(b.title))
```

But I'll leave it to you to decide if that's too concise/clever.

Reply:
No, this is super helpful; TIL about both `localeCompare` and the `slice()` trick to copy an array 💫